spec/models/character_spec.rb

```diff
@@ -2,6 +2,24 @@
 
 RSpec.describe Character, type: :model do
   describe 'validations' do
-    it { should validate_presence_of(:name) }
+    it { should validate_presence_of :name }
+    it { should validate_presence_of :intelligence }
+    it { should validate_presence_of :strength }
+    it { should validate_presence_of :speed }
+    it { should validate_presence_of :durability }
+    it { should validate_presence_of :power }
```

Comment:
If we come into an issue where some attributes are blank we will have to revisit this and account for blank and new.  Nice job on all this!